frontend/packages/shipwright-plugin/integration-tests/features/e2e/shipwright-ci.feature

```diff
@@ -4,14 +4,14 @@ Feature: Shipwright build details page
 
         Background:
             Given user has installed OpenShift Pipelines Operator
-              And user is at developer perspective
-              And user has created or selected namespace "aut-shipwright-build-details"
-              And user has installed Shipwright Operator
-              And user is at Add page
-              And user has created shipwright builds
+            And user is at developer perspective
+            And user has created or selected namespace "aut-shipwright-build-details"
+            And user has installed Shipwright Operator
+            And user is at Add page
+            And user has created shipwright builds
 
 
-        @smoke
+        @smoke @manual
```

Comment:
What is the need to mark these test cases as manual?

Reply:
Have removed them :+1: